lib/rubocop/cop/rails/duration_arithmetic.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks if a duration is added to or subtracted from `Time.current`.
+      #
+      # @example
+      #   # bad
+      #   Time.current - 1.minute
+      #   Time.current + 2.days
+      #
+      #   # good - using relative would make it harder to express and read
+      #   Date.yesterday + 3.days
+      #   created_at - 1.minute
+      #   3.days - 1.hour
+      #
+      #   # good
+      #   1.minute.ago
+      #   2.days.from_now
+      class DurationArithmetic < Base
+        extend AutoCorrector
+
+        MSG = 'Do not add or subtract duration.'
+
+        RESTRICT_ON_SEND = %i[+ -].freeze
+
+        DURATIONS = Set[:second, :seconds, :minute, :minutes, :hour, :hours,
```

Comment:
I'm curious if there was a specific reason that `month`, `months`, `year`, and `years` were excluded from these duration methods.

Reply:
I've only checked [Numeric](https://api.rubyonrails.org/classes/Numeric.html) and completely forgot about [Integer](https://api.rubyonrails.org/classes/Integer.html).

Would you like to send a PR to fix this?

A spec to check that all are covered would be nice to have, too.

Reply:
@pirj done! #622